tests/qactivation_test.py

```diff
@@ -23,12 +23,14 @@
 import pytest
 from tensorflow.keras import backend as K
 
+import qkeras
```

Comment:
Please replace this with `from google3.third_party.qkeras import set_internal_sigmoid` since it follows the same pattern as the other imports

Reply:
WIth this change the test now fails with error:`E   ModuleNotFoundError: No module named 'google3'`

Reply:
hi @danielemoro we do not need to add "google3.third_party" in our github version. It will be handled by our copybara service.

Reply:
My bad. I meant to write `from qkeras import set_internal_sigmoid` since all of the other import statements import a specific module from qkeras rather than the entirety of qkeras.